config/routes.rb

```diff
@@ -307,6 +307,7 @@
       post '/personal_key' => 'personal_key#update'
       get '/forgot_password' => 'forgot_password#new'
       post '/forgot_password' => 'forgot_password#update'
+      get '/document_capture' => 'document_capture#show'
```

Comment:
Do we need a `PUT` route here? Right now submitting the doc capture form 404s. (Or is that a different PR?)

Reply:
AH looks like this 9098, apologies for the extra noise here.